core/src/main/java/tc/oc/pgm/score/ScoreMatchModule.java

```diff
@@ -97,6 +104,10 @@ public double getScore(Competitor competitor) {
     return this.scores.get(competitor);
   }
 
+  public int getDefaultScoreLimit() {
+    return defaultScore;
+  }
+
```

Comment:
```suggestion
```

Don't expose this, it's an implementation detail that shouldn't be known by outside classes